src/screens/Followings/FollowersScreen.js

```diff
@@ -37,12 +41,16 @@
   };
 
   React.useEffect(() => {
-    if (searchText.length === 0) {
+    if (!isFirstTimeOpen && searchText.length > 0) {
       fetchFollower(true, searchText);
     }
   }, [searchText]);
 
   React.useEffect(() => {
+    if (isFirstTimeOpen && !params?.params?.initialData) {
+      fetchFollower(true, '');
+    }
+    setIsFirstTimeOpen(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional check for `isFirstTimeOpen` and `searchText.length > 0` before calling `fetchFollower` is a good practice to prevent unnecessary API calls. However, make sure that `fetchFollower` can handle an empty string as a parameter. Also, the second useEffect hook now depends on `isFirstTimeOpen` and `params?.params?.initialData`. Ensure that these dependencies are managed properly.
```diff
+ if (!isFirstTimeOpen && searchText.length > 0) {
+   fetchFollower(true, searchText);
+ }
+ if (isFirstTimeOpen && !params?.params?.initialData) {
+   fetchFollower(true, '');
+ }
+ setIsFirstTimeOpen(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->